packages/components/package.json

```diff
@@ -11,42 +11,28 @@
     "./style.css": "./style.css",
     "./package.json": "./package.json",
     ".": {
-      "import": "./dist/index.js",
-      "types": "./dist/index.d.mts"
+      "types": "./dist/index.d.mts",
+      "import": "./dist/index.js"
     },
     "./next.config": {
-      "import": "./dist/next.config.mjs",
-      "types": "./dist/next.config.d.mts"
+      "types": "./dist/server/next.config.d.mts",
+      "import": "./dist/server/next.config.js",
+      "require": "./dist/server/next.config.js"
     },
-    "./compile": {
-      "import": "./dist/compile.mjs",
-      "types": "./dist/compile.d.mts"
+    "./server": {
+      "types": "./dist/server/index.d.mts",
+      "import": "./dist/server/index.js"
+    },
+    "./pages": {
+      "types": "./dist/server/pages.d.mts",
+      "import": "./dist/server/pages.js"
     },
     "./*": {
-      "import": "./dist/*.js",
-      "types": "./dist/*.d.mts"
+      "types": "./dist/*.d.mts",
+      "import": "./dist/*.js"
     }
   },
   "types": "./dist/index.d.mts",
-  "typesVersions": {
-    "*": {
-      ".": [
-        "./dist/index.d.mts"
-      ],
-      "logos": [
-        "./dist/logos.d.mts"
-      ],
-      "products": [
-        "./dist/products.d.mts"
-      ],
-      "next.config": [
-        "./dist/next.config.d.mts"
-      ],
-      "compile": [
-        "./dist/compile.d.mts"
-      ]
-    }
-  },
```

Comment:
removing this requires updating `tsconfig.json` to set `moduleResolution: bundler`